doc/modules/ensemble.rst

```diff
@@ -1033,3 +1032,90 @@ The following example shows how to fit the VotingRegressor::
 .. topic:: Examples:
 
   * :ref:`sphx_glr_auto_examples_ensemble_plot_voting_regressor.py`
+
+.. _stacking:
+
+Stacked generalization
+----------------------
+
+Stacked generalization is a method for combining estimators to reduce their
+biases [W1992]. More precisely, the predictions of each individual estimator
+are stacked together and used by another estimator to compute the final
```

Comment:
I think we can make the "stacking" procedure a bit clearer with e.g.

```suggestion
are stacked together and used as input to a final estimator to compute the final
```

Before I looked at the code I thought the final estimator was learning some weights or something but it's more trivial than that.